### ISSUE TITLE
All buttons should have a descriptive text

### ISSUE THREAD

**Issue by mtelgkamp** [neos/neos-ui#1592]
### Description

For better accessibility all buttons should have a descriptive text. Either the text of the button or an aria-label.

#### Expected behavior

Screenreaders should announce the function of a button

#### Actual behavior

Some buttons are announced as "button" only

- PublishDropDown > ThemedDropDownWrapper
  - [x] ThemedIcon
- SecondaryToolbar
  - [x] PreviewButton
  - [x] FullScreenButton
- NodeTreeToolBar 
  - [x] ToggleContentTree
